### ISSUE TITLE
Plugins settings are broken after upgrading to 0.17.15

### ISSUE THREAD

**Issue by mhdali** [vrtmrz/obsidian-livesync#164]
After upgrading to 0.17.15 plugins dialog doesn't show any settings from other hosts, nor its plugins (screenshot attached). Feel free to let me know if you need any more information.

![2023-01-23_10:40:34](https://user-images.githubusercontent.com/6121579/213957863-b4d9732c-0793-4d56-90be-62bed79689dd.jpg)

Debug info:
```
SYSTEM INFO:
	Obsidian version: v1.1.9
	Installer version: v1.1.9
	Operating system: #1 SMP PREEMPT_DYNAMIC Wed, 18 Jan 2023 19:54:38 +0000 XXX
	Login status: not logged in
	Insider build toggle: off
	Live preview: on
	Legacy editor: off
	Base theme: dark
	Community theme: none
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 12
	Plugins enabled: 9
		1: XXX
		2: XXX
		3: XXX
		4: XXX
		5: XXX
		6: XXX
		7: XXX
		8: Self-hosted LiveSync v0.17.15
		9: XXX

RECOMMENDATIONS:
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.
```

**Comment by vrtmrz** [comment]
Thank you for reporting this! This problem has been reproduced! 
It could be fixed immediately, but I am integrating this feature into the hidden file sync. 

The code is already almost written, Sorry for your inconvenience, please wait a few moments while the testing is completed.

**Comment by vrtmrz** [comment]
Sorry for being late to notify this! 
This issue has been fixed at v0.17.16. I would like to thank you for your reporting again!
Could you please try this?

**Comment by mhdali** [comment]
Yes this issue has been resolved after upgrading to 0.17.18
